docker/prod/docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+version: '3'
+
+volumes:
+    postgres_data: {}
+    postgres_backup: {}
+    static_dir: {}
+
+services:
+  application:
+    container_name: prod_andela_socials
+    build:
+      context: ../../
+      dockerfile: docker/prod/Dockerfile
+    env_file:
+      - ../../.env
+    ports:
+      - "8000:8000"
+    depends_on:
+      - database
+    volumes:
+      - static_dir:/andela_social/staticfiles
+
+  database:
+    container_name: prod_andela_socials_db
+    build:
+      context: ../../docker/database
+      dockerfile: Dockerfile
+    restart: always
+    env_file:
+      - ../../.env
+    volumes:
+      - postgres_data:/var/lib/postgresql/data
+      - postgres_backup:/backups
+    ports:
+      - "5432"
```

Comment:
What is this mapping to, recall this compose file is for reference locally.

Reply:
Ideally, it should map to 5432 on the host. But then I decided to just specify for the container so that an [ephemeral port is chosen for the host](https://docs.docker.com/compose/compose-file/compose-file-v2/#ports) because of this error I encountered; 
```
ERROR: for database  Cannot start service database: driver failed programming external connectivity on endpoint dev_andela_social_db
 (266eb6f54d0899013a654fa348dafadc9279a93a80c51c294f8c89654df5909d): Error starting userland proxy: Bind for 0.0.0.0:5432 failed: port is already allocated
```
This was due to the postgresql service on my PC already using 5432.
I had two choices; to stop my running postgresql service or choose another port for it.
I went with the second option because I felt most users would also have postgresql or any other database service running locally on port 5432. Saving them the trouble would make setup easier.
For cases of debugging or others that require knowing the mapped port, `docker container ls` would show all running containers with their ports. I believe that would suffice. Any further insights will be appreciated

Reply:
Explicit is better than implicit, choosing an alternate port for this should suffice. This should not be a serious issue though, you should map another port to 5432. eg `8092: 5432`

Reply:
Thanks, I will make the updates